coralproject/talk
# Final copy update for DSA notifications

## What does this PR do?



## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags?

No

## If any indexes were added, were they added to `INDEXES.md`?

No new indices

## How do I test this PR?

- enable DSA reporting flow via Admin > Config > General > DSA
- create a comment with a commenter
- report it with an illegal report (DSA report) as the same commenter
- use an admin/mod to process the DSA report as illegal
- check the commenter's stream side notifications
    - should have a notification about the comment being rejected
    - should have a notification about their DSA report being decided/reviewed
- create another comment as commenter
- have an admin/mod reject it normally via mod queue while selecting a reason
- check the notifications for the commenter
    - should have a notification about the comment being rejected

## Where any tests migrated to React Testing Library?

No

## How do we deploy this PR?

Merge into DSA epic branch and release with rest of DSA features.
